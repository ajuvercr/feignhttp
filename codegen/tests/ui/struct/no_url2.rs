use feignhttp_codegen::feign;

// error: metadata url not specified
//    |  #[feign(path = "http://xxx")]
//    |  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

struct Http;

#[feign(path = "http://xxx")]
impl Http {}

fn main() {}
