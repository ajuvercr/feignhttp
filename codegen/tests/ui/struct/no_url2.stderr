error: metadata url not specified
 --> tests/ui/struct/no_url2.rs:9:1
  |
9 | #[feign(path = "http://xxx")]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `feign` (in Nightly builds, run with -Z macro-backtrace for more info)
